common/changes/@bentley/context-registry-client/spencer-context-registry-abstraction_2021-08-18-18-55.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/context-registry-client",
+      "comment": "Replaced ContextRegistry with generalized ITwinAccess interface",
+      "type": "none"
```

Comment:
Why is this marked as "none" type change ?

Reply:
this repo isn't setup to actually use the "type" in the rush change since we do everything in lock step.

Reply:
In core, since all the packages are in lockstep, the type here doesn't matter, as all packages will be incremented together.